CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+### Deprecated
```

Comment:
this should be a `### Breaking Changes` header if we are actually removing support. see: https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md for more info

Reply:
Done.